data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -39,23 +39,20 @@ spec:
             - /bin/bash
             - -ce
             - |
-              echo 'Installing bridge and tuning CNIs'
-              cni_mount_dir=/opt/cni/bin
-              sourcebinpath=/usr/src/github.com/containernetworking/plugins/bin
-              cp --remove-destination ${sourcebinpath}/bridge ${cni_mount_dir}/cnv-bridge
-              cp --remove-destination ${sourcebinpath}/tuning ${cni_mount_dir}/cnv-tuning
+              if find ${cni_mount_dir}/bridge &>/dev/null; then
+                ln -s ${cni_mount_dir}/bridge ${cni_mount_dir}/cnv-bridge
+                ln -s ${cni_mount_dir}/tuning ${cni_mount_dir}/cnv-tuning
+              else
+                cni_mount_dir=/opt/cni/bin
+                sourcebinpath=/usr/src/github.com/containernetworking/plugins/bin
+                cp --remove-destination ${sourcebinpath}/bridge ${cni_mount_dir}/cnv-bridge
+                cp --remove-destination ${sourcebinpath}/tuning ${cni_mount_dir}/cnv-tuning
```

Comment:
We still need to deploy the `bridge` and `tuning` binaries as we used to. Otherwise KubeVirt e2e tests will start failing and users won't be able to use their old NetworkAttachmentDefinitions. To maintain the existing behavior, we still need to install `bridge` and `cnv-bridge` on non-OpenShift environments